src/rcore.c

```diff
@@ -959,7 +959,7 @@ void EndDrawing(void)
         CORE.Time.frame += waitTime;    // Total frame time: update + draw + wait
     }
 
-    PollInputEvents();      // Poll user events (before next frame update)
+//    PollInputEvents();      // Poll user events (before next frame update)
```

Comment:
Let's just do `#ifndef PLATFORM_COMMA` for now.

In the future, we should just define `SUPPORT_CUSTOM_FRAME_CONTROL` and implement all this on our side